src/proto/ProtoHeaderWrap.cc

```diff
@@ -59,7 +63,7 @@ std::string ProtoHeaderWrap::wrap(std::string& protoHeaderString) {
 }
 
 bool ProtoHeaderWrap::unwrap(std::shared_ptr<WorkerResponse>& response, std::vector<char>& buffer) {
-    response->headerSize = static_cast<unsigned char>(buffer[0]);
+    response->headerSize = static_cast<unsigned char const>(buffer[0]);
```

Comment:
Does it hurt? Changing the contents of the buffer would be bad.

Reply:
It does not do anything, and it's not preventing any potential changes to buffer (because there could not be any changes in this statement). It's sort of thing like this:

    int a = 1;
    int b = static_cast<int const>(a);

which is legal but pointless.